webpack.config.js

```diff
@@ -39,8 +39,8 @@ const extractConfig = {
 const entryPointNames = [
 	'element',
 	'i18n',
-	'components',
 	'utils',
+	'components',
```

Comment:
At one point, this was important to get right in the webpack config because otherwise it would break the test build.  It seems this is no longer the case since we moved to Jest.  I'll update accordingly.